fly/commands/ordering_pipeline.go

```diff
@@ -12,7 +13,8 @@ import (
 var ErrMissingPipelineName = errors.New("Need to specify atleast one pipeline name")
 
 type OrderPipelinesCommand struct {
-	Pipelines []flaghelpers.PipelineFlag `short:"p" long:"pipeline" required:"true" description:"Name of pipeline to order"`
+	All       bool                       `short:"a"  long:"all" description:"Order all pipelines alphabetically"`
```

Comment:
Maybe the flag can be called "Alphabetical"?

Having the flag be called "all" might be a little confusing, since it would read "fly order-pipelines --all" which is hard to know what the actual purpose of the command, which is to order the pipeline alphabetically. @vito it might be more of a question for you though.